src/popup/utils/constants.js

```diff
@@ -64,32 +64,35 @@ export const calculateFee = (type, params) => {
   };
 };
 
+export const defaultNetworks = [
+  {
+    url: 'https://testnet.aeternity.io',
+    internalUrl: 'https://testnet.aeternity.io',
+    networkId: 'ae_uat',
+    middlewareUrl: 'https://testnet.aeternity.io',
+    explorerUrl: 'https://testnet.aeternity.io',
+    compilerUrl: 'https://latest.compiler.aepps.com',
+    tokenRegistry: 'ct_UAzV9RcXEMsFcUCmrPN4iphbZroM7EHk3wvdidDYgZGGBo3hV',
+    tokenRegistryLima: 'ct_Dnwribmd21YrxSQnqXCB5vTFPrgYJx2eg2TrbLvbdyEbTMejw',
+    tipContract: 'ct_YpQpntd6fi6r3VXnGW7vJiwPYtiKvutUDY35L4PiqkbKEVRqj',
+    name: 'Testnet',
+  },
+  {
+    url: 'https://mainnet.aeternity.io',
+    internalUrl: 'https://mainnet.aeternity.io',
+    networkId: 'ae_mainnet',
+    middlewareUrl: 'https://mainnet.aeternity.io',
+    explorerUrl: 'https://mainnet.aeternity.io',
```

Comment:
```suggestion
    middlewareUrl: 'https://mainnet.aeternal.io',
    explorerUrl: 'https://mainnet.aeternal.io',
```


We still use the old middleware instance. There is no middleware hosted on the URL you suggest here.